vpp/isisx/vpp2202/isisx/isisx_api.c

```diff
@@ -44,7 +45,7 @@ vl_api_isisx_plugin_get_version_t_handler (vl_api_isisx_plugin_get_version_t * m
 
   rmp = vl_msg_api_alloc (sizeof (*rmp));
   rmp->_vl_msg_id =
-    ntohs (ISISX_MSG_ID (VL_API_ISISX_PLUGIN_GET_VERSION_REPLY));
+    ntohs (VL_API_ISISX_PLUGIN_GET_VERSION_REPLY + REPLY_MSG_ID_BASE);
```

Comment:
The only suggestion I have is that it would be possible to throw out few lines of code and replace them with a macro.
```
static void
vl_api_isisx_plugin_get_version_t_handler (
  vl_api_isisx_plugin_get_version_t *mp)
{
  vl_api_isisx_plugin_get_version_reply_t *rmp;
  REPLY_MACRO_DETAILS2 (VL_API_ISISX_PLUGIN_GET_VERSION_REPLY, ({
			  rmp->major = htonl (ISISX_PLUGIN_VERSION_MAJOR);
			  rmp->minor = htonl (ISISX_PLUGIN_VERSION_MINOR);
			}));
}
```